azure/scope/cluster.go

```diff
@@ -128,12 +128,19 @@ func (s *ClusterScope) PublicIPSpecs() []azure.PublicIPSpec {
 		}
 	}
 
-	return []azure.PublicIPSpec{
-		controlPlaneOutboundIP,
-		{
-			Name: azure.GenerateNodeOutboundIPName(s.ClusterName()),
-		},
+	var publicIPSpec []azure.PublicIPSpec
```

Comment:
`publicIPSpecs` instead of `publicIPSpec`

Reply:
Replaced.